nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -120,31 +118,31 @@ in
               path /api/* /pictrs/* feeds/* nodeinfo/*
             }
             handle @for_backend {
-              reverse_proxy 127.0.0.1:${toString cfg.settings.port}
+              reverse_proxy 0.0.0.0:${toString cfg.settings.port}
```

Comment:
This should stay on 127.0.0.1 because we want to connect over localhost. I am not sure if that even would work.